packages/google_maps_flutter/google_maps_flutter/lib/src/google_map.dart

```diff
@@ -547,6 +562,8 @@ class _GoogleMapOptions {
         buildingsEnabled = map.buildingsEnabled,
         assert(!map.liteModeEnabled || Platform.isAndroid);
 
+  final GestureHandling? gestureHandling;
```

Comment:
Since this is web-only, and it doesn't seem to have a direct equivalent on android/iOS (they use booleans there, and the `gestureRecognizers` are more a flutter thing), let's call this:

`WebGestureHandling? webGestureHandling`

So it's very clear that it's a web-only setting that won't work elsewhere.

Reply:
Done